.gitattributes

```diff
@@ -1,8 +1,8 @@
 **/generated-archives/*.tar* filter=lfs-disabled diff=lfs merge=lfs -text
 
 # assure line feeds don't interfere with our working copy hash
-**/tests/fixtures/**/*.sh text crlf=input	eol=lf
```

Comment:
A great catch! I also stumbled over it some time ago but was too afraid to try it 😅.

Reply:
It might be enough to further simplify this by removing `crlf=input` and just having the pattern and `eol=lf`. I'm unaccustomed to using `crlf=input` and it looks like, today, it might actually be equivalent to `eol=lf` and preserved for [backward compatibility](https://git-scm.com/docs/gitattributes#_backwards_compatibility_with_crlf_attribute).

Reply:
Yes, that's probably possible and `gitoxide` should certainly try to be a better example :D.

Reply:
Among other changes, I've removed `crlf=input` from both `.gitattributes` files that had it, in #169. (They both also had `eol=lf`.)